scripts/west_commands/patch.py

```diff
@@ -181,6 +192,9 @@ def filter_args(self, args):
             args.patch_yml = manifest_dir / args.patch_yml.relative_to(_WEST_MANIFEST_DIR)
         if args.west_workspace.is_relative_to(_WEST_TOPDIR):
             args.west_workspace = topdir / args.west_workspace.relative_to(_WEST_TOPDIR)
+        if args.modules:
+            # Remove trailing slashes
+            args.modules = [m.rstrip('/') for m in args.modules]
```

Comment:
Would it make sense to create a Path object out of mod instead of calling rstrip?

Maybe even creating path objects directly through the argument parser.